### DISCUSSION TITLE
Queria deixar meu agradecimentos aos criados e mantenedores deste repositório.

### DISCUSSION THREAD

**Discussion by marcelofabianov** [laradumps/laradumps#99] [General]
Obrigado a todas as pessoas que estão envolvidas ou ja se envolveram neste repositório!

Quero agradecer a cada um por deixar minha vida de desenvolvedor melhor !

OBRIGADO !

**Comment by dansysanalyst** [comment]
Obrigado @marcelofabianov!!! Muito bacana sua atitude, viva a comunidade Laravel!

O LaraDumps está aberto a contribuições. Sinta-se mais que bem-vindo a participar do projeto!
Vale ressaltar que há várias maneiras colaborar além de enviar código. Um tweet, artigo no blog/linkedin, recomendações em grupos de dev. Tudo isso conta e muito!

Um abraço,

PS: Vou converter para "conversation" para sempre ter a mensagem disponível por aqui!

**Comment by vitorhugosg** [comment]
Pow foi mal criei uma issue, mas vou falar aqui, mano perfeito! parabéns a toda equipe!
